CONTRIBUTING.md

```diff
@@ -35,6 +36,11 @@ You can submit your changes to ASReview [by creating a pull request](https://git
 - Add a new dataset to [our open dataset repository](https://github.com/asreview/systematic-review-datasets/pull/11)
   - See this [pull request](https://github.com/asreview/systematic-review-datasets/pull/11)) for an example.
 
+## Discussions
+Got any questions for the ASReview community?
+Or perhaps you know the answer to a question asked? 💡
+Please don't hesitate to post on our [discussion board](https://github.com/asreview/asreview/discussions).
```

Comment:
Usually, we try to avoid "please" and "our". But I see it is used a lot in this page. Interested in changing this?